src/components/calculator/Calculator.css

```diff
@@ -0,0 +1,36 @@
+.calculator {
+  width: 300px;
+  margin: 10px auto;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+}
+
+.output {
+  font-size: 24px;
+  text-align: right;
+  margin-bottom: 10px;
+  padding: 10px;
+  background-color: #f4f4f4;
+  border: 1px solid #ccc;
+  border-radius: 3px;
+}
+
+.buttons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 10px;
+}
+
+button {
+  font-size: 18px;
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 3px;
+  cursor: pointer;
```

Comment:
- [ ] [REQUIRED] Good job so far! The `Calculator.js` component should match the following given `Microverse` design, please implement it 🙏🏼 

![image](https://github.com/egichuhis/math-magicians/assets/63915024/a905ba79-4a1d-4ea7-a88b-a98a50a7d2ea)